resources/sts/4.12/hypershift/ControlPlaneOperatorPolicy.json

```diff
@@ -2,32 +2,89 @@
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Effect": "Allow",
             "Action": [
-                "ec2:CreateVpcEndpoint",
                 "ec2:DescribeVpcEndpoints",
-                "ec2:ModifyVpcEndpoint",
-                "ec2:DeleteVpcEndpoints",
-                "ec2:CreateTags",
-                "route53:ListHostedZones",
-                "ec2:CreateSecurityGroup",
-                "ec2:AuthorizeSecurityGroupIngress",
-                "ec2:AuthorizeSecurityGroupEgress",
-                "ec2:DeleteSecurityGroup",
-                "ec2:RevokeSecurityGroupIngress",
-                "ec2:RevokeSecurityGroupEgress",
-                "ec2:DescribeSecurityGroups",
-                "ec2:DescribeVpcs"
+                "route53:ListHostedZones"
             ],
+            "Effect": "Allow",
             "Resource": "*"
         },
         {
-            "Effect": "Allow",
             "Action": [
                 "route53:ChangeResourceRecordSets",
                 "route53:ListResourceRecordSets"
             ],
-            "Resource": "arn:aws:route53:::%s"
+            "Effect": "Allow",
+            "Resource": [
```

Comment:
Same thought here, can these be limited to the hosted zones with the right URLs?

Reply:
good point, going to test this before committing. Thanks

Reply:
this has been tested and fixed